docs/quick-starts/03-quick-start-cloud-run.md

```diff
@@ -27,15 +30,72 @@ Make sure that billing is enabled for your project ([see how]).
 Install and configure [gcloud].
 
 If you need help installing or configuring `gcloud` on your machine, see
-[Quickstart: Getting started with Cloud SDK][quickstart].
+[Quickstart: Getting started with Cloud SDK][quickstart]. Don't forget to run
+the `gcloud auth login` command to authenticate your terminal.
 
 > Note: `gcloud` is preinstalled in [Cloud Shell].
 
+### Set your gcloud project
+
+Update your `gcloud` configuration to use your project ID.
+
+```shell
+$ gcloud config set core/project <PROJECT-ID>
+Updated property [core/project].
+```
+
 ## Get a copy of the `hello` example
 
 Clone this repo or download a [zip] archive and extract the contents. Change
 directory to `examples/hello`.
 
+## Build and deploy with a single command
+
+The next two sections cover two commands that can be replaced with a single
```

Comment:
Maybe something like

```suggestion
The next two sections cover one command (now in beta) that can replace two commands.
```

Reply:
Hmm that's not better either, but agree this should and can be reworded to read more smoothly. Gimme a moment...

Reply:
How's this?